BGL/include/CGAL/boost/graph/IO/Generic_facegraph_printer.h

```diff
@@ -143,7 +143,7 @@ class Generic_facegraph_printer
                           has_vertex_textures                 );
 
     vertices_size_type id = 0;
-    for(const vertex_descriptor v : vertices(g))
+    for(const vertex_descriptor& v : vertices(g))
```

Comment:
Actually, I included that in this PR because my platform was complaining and having warnings related to this when I was compiling my examples. But I can remove it, too. Maybe it is better to propose a little PR related only to this change.